blueprints/ember-electron/index.js

```diff
@@ -24,96 +21,63 @@ class EmberElectronBlueprint extends Blueprint {
   afterInstall(/* options */) {
     let logger = new Logger(this);
 
-    return this._installDependencies(logger)
-      .then(() => this._createResourcesDirectories())
-      .then(() => this._ensurePackageJsonConfiguration())
-      .then(() => this._updateGitignore())
-      .then(() => {
-        let configMessage = 'Ember Electron requires configuration. Please consult the Readme to ensure that this addon works!';
-        logger.message(configMessage, logger.chalk.yellow);
-        logger.message('https://github.com/felixrieseberg/ember-electron');
-      });
+    return this._installElectronTooling(logger)
+      .then(() => this._createResourcesDirectories(logger))
+      .then(() => this._ensurePackageJsonConfiguration(logger));
   }
 
-  _installDependencies(logger) {
-    // Since addPackagesToProject() doesn't give us fine-grained control over save vs. save-dev,
-    // or the exact flag, we use:
-    let task = this.taskFor('npm-install');
-
-    logger.startProgress('Installing electron tools (electron-forge + electron-protocol-serve)');
-
-    // There are two parts of electron-forge's import command that are relevant to us.
-    // One is installing dependencies, and the other is setting up the electron-forge configuration.
-    // Rather than embedding the forge configuration inline in package.json
-    // we want it in an external file, so we install the dependencies here ourselves
-    // (including our own dependencies), and add an entry to package.json
-    // pointing the forge config to the config file that is copied from the blueprint files.
-
-    const { devDeps, exactDevDeps, deps } = require('electron-forge/dist/init/init-npm');
-
-    return task.run({
-      'save-dev': true,
-      verbose: false,
-      packages: devDeps
-    }).then(() => task.run({
-      // Dependencies we need to keep locked at specific versions
-      'save-dev': true,
-      'save-exact': true,
-      verbose: false,
-      packages: exactDevDeps
-    })).then(() => task.run({
-      // Production dependencies
-      save: true,
-      verbose: true,
-      packages: ['electron-protocol-serve@1.1.0', ...deps]
-    }));
+  _installElectronTooling(logger) {
+    logger.startProgress('Installing electron build tools');
+
+    return efImport({
+      updateScripts: false,
+      outDir: 'electron-out',
+    })
+      .then(() => this.addPackageToProject('devtron', '^1.4.0'))
```

Comment:
I was on the fence about this too, but ultimately opted for the "use available upstream tooling" option (again, since if we're working around it we ought to focus on landing it upstream instead).

I think the brief `// n.b. ...` note below explains why the change in form, and would prefer leaving this as is. Happy to wait on another voice here if you consider this strong dissent / a blocker.